packages/material-ui/src/StepLabel/StepLabel.js

```diff
@@ -184,6 +195,18 @@ StepLabel.propTypes = {
    * @ignore
    */
   orientation: PropTypes.oneOf(['horizontal', 'vertical']),
+  /**
+   * The React Element to render in place of the [`StepIcon`](/api/step-icon) element.
+   */
+  StepIconComponent: PropTypes.oneOfType([
```

Comment:
You expect a component here? string, object, function are sufficient in that case.

Reply:
ok thanks